docs/development/README.md

```diff
@@ -25,23 +25,49 @@ inside a container.
 
 If you do not receive a prompt, or when you feel like starting from a fresh environment:
 
-1. `Ctrl/Cmd+Shift+P` to bring up the command palette in Visual Studio Code
+1. <kbd>Ctrl</kbd>/<kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>P</kbd> to bring up the command palette in Visual Studio Code
 1. Type `Remote-Containers` to filter the commands
 1. Select `Rebuild and Reopen in Container` to completely rebuild the devcontainer
 1. Select `Reopen in Container` to reopen the most recent devcontainer build
 
 ## Attach a debugger
 
-Once running inside a container, press **`F5`** to attach a debugger to the client, running on `http://localhost` at a port
+Once running inside a container, press <kbd>F5</kbd> to attach a debugger to the client, running on `http://localhost` at a port
 dynamically assigned by Docker. See [Docker dynamic ports](./docker-dynamic-ports.md) for more information.
 
-Add breakpoints in the code and browse the local site to trigger a pause. Press `F5` to continue execution from the breakpoint.
+Add breakpoints in the code and browse the local site to trigger a pause. Press <kbd>F5</kbd> to continue execution from the breakpoint.
+
+### Changing launch configuration
+
+By default, the application is launched with `DJANGO_DEBUG=True`, causing Django to provide additional logging and error output and to relax certain security settings.
+
+Alternatively, you may attach to an instance launched with `DJANGO_DEBUG=False`, to allow debugging the app in a state more similar to production.
+
+In VS Code, press <kbd>Ctrl</kbd>/<kbd>Cmd</kbd> + <kbd>Shift</kbd> + <kbd>D</kbd> to open the `Run and Debug` pane, where you can select between the various configurations (disregard the duplicate entry, selecting either will work):
+
+![Screenshot of the VSCode Run and Debug pane, showing selection of the launch configuration](img/vscode-debugger-launch-config.png)
+
+The [environment](../configuration/environment-variables.md) can also be overridden for the debug session by editing the configuration in `.vscode/launch.json`, where any of the supported environment variables may be specified in the `env` block:
```

Comment:
To be extra explicit and search-friendly, add the example of `For example, to test the app with reCAPTCHA environment variables...` and then link to https://github.com/cal-itp/benefits/pull/1071 

Otherwise LGTM 👍

Reply:
Added in [0d0cd61](https://github.com/cal-itp/benefits/pull/1160/commits/0d0cd618837552960b6aff5ab638f82343b895b6)